ReactWindows/ReactNative.Shared/UIManager/Events/Event.cs

```diff
@@ -94,6 +96,18 @@ public virtual short CoalescingKey
         }
 
         /// <summary>
+        /// A key used to guarantee that two events with relative sequencing
+        /// requirements are dispatched in order.
+        /// </summary>
+        public virtual int SortingKey
+        {
+            get
+            {
+                return 0;
```

Comment:
Just staying consistent with the rest of the class. At some point, if we decide that's the preferred style, we can switch to auto properties everywhere